autoPyTorch/api/base_task.py

```diff
@@ -1245,17 +1239,32 @@ def __del__(self) -> None:
         # until the estimator is deleted
         self._backend.context.delete_directories(force=False)
 
-    @typing.no_type_check
     def get_incumbent_results(
-            self
-    ):
-        pass
+        self,
+        include_traditional: bool = False
+    ) -> Tuple[Configuration, Dict]:
```

Comment:
This should be a Dict[str, Union[str, int, float]]?

Reply:
if we want to be more specific, then yes. I'll do it

Reply:
Thanks, it is not only being specific but if not specified, it is converted to `Dict[Any, Any]` which basically checks almost nothing :)